lib/lhm/migrator.rb

```diff
@@ -193,16 +193,19 @@ def validate
 
       dest = @origin.destination_name
 
-      if @connection.table_exists?(dest)
+      if @connection.table_exists?(dest) && ENV['LHM_RESUME_AT'].nil?
         error("#{ dest } should not exist; not cleaned up from previous run?")
       end
     end
 
     def execute
-      destination_create
-      @statements.each do |stmt|
-        @connection.execute(tagged(stmt))
+      unless ENV['LHM_RESUME_AT'].present?
```

Comment:
💭 I have a slight preference for `if` where possible
```suggestion
      if ENV['LHM_RESUME_AT'].blank?
```